### ISSUE TITLE
Implement character rotation

### ISSUE THREAD

**Issue by Jean-M22** [radubig/FMI-IPJC#4]
Create a simple character rotation script which takes in the cursor position and rotates a player to face it. 
Bonus points if you also check for the active input device and make it compatible with the right joystick on gamepads. 
You should use Unity's NEW Input System.
 
You should make a prefab that contains an example character with the relevant scripts attached.

Some resources I found on mouse position;
https://www.youtube.com/watch?v=JfpMIUDa-Mk

Some resources I found on the new input system:
[youtube.com/watch?v=Pzd8NhcRzVo](https://www.youtube.com/watch?v=Pzd8NhcRzVo)
[youtube.com/watch?v=HmXU4dZbaMw](https://www.youtube.com/watch?v=HmXU4dZbaMw)

---

This issue could be done after #3, however it doesn't depend on it and in all fairness can be completed separately (and then the two players can be merged into one)
